React/React.xcodeproj/project.pbxproj

```diff
@@ -4862,6 +4862,13 @@
 			buildSettings = {
 				CLANG_CXX_LANGUAGE_STANDARD = "c++14";
 				CLANG_STATIC_ANALYZER_MODE = deep;
+				GCC_PREPROCESSOR_DEFINITIONS = (
```

Comment:
Why do we need this?

Reply:
The RCT_METRO_PORT is passed in as an environment variable to keep the code as clean as possible.  I only added the RCT_METRO_PORT, when adding the GCC preprocessor definitions xcode pre-populated the list with the other defines.